tests/test_resource.py

```diff
@@ -1,92 +1,92 @@
-from __future__ import absolute_import, unicode_literals
```

Comment:
You might need to check your IDE. There is no real changes in this file

Reply:
Yes this is an IDE/OS issue. I've reverted the file.